### PR TITLE
more secure solution in comment block

### PR DESCRIPTION
**Pull request by manonthemat**

### CODE REVIEW

File: exercises/uploads/solution/solution.js [hunk 1]
```diff
@@ -1,45 +1,37 @@
...
-
+var Hapi = require("hapi");
 
 var server = new Hapi.Server();
 
 server.connection({
-    host: 'localhost',
-    port: Number(process.argv[2] || 8080)
+  port: Number(process.argv[2] || 8080),
+  host: "localhost"
```
Comment (nvcexploder) [inline exercises/uploads/solution/solution.js]: I'd prefer these stay single quotes throughout.

**Comment by nvcexploder** [comment]
Hey! Thanks! This looks great! There's a few places where single quotes were replaced with double quotes. I'd like them to stay single quotes - as soon as that's changed I'll merge it in! :+1:

**Comment by manonthemat** [comment]
It's my fault, I gave in to the linter that kept screaming at me for using single quotes instead of double quotes. But I agree that there should be single quotes since that's how everybody else is doing it in this project. I'll change it later today. :+1:

**Comment by nvcexploder** [comment]
Thanks!

**Comment by manonthemat** [comment]
Just checked again. With the last two commits I should've gotten them all.

**Comment by nvcexploder** [comment]
looks good! Thanks!

**Pull request merged**